fritzconnection/core/fritzconnection.py

```diff
@@ -75,7 +82,8 @@ def __init__(
         password=None,
         timeout=None,
         use_tls=False,
-        concurrent_connections=10,
+        pool_connections=DEFAULT_POOL_CONNECTIONS,
+        pool_maxsize=DEFAULT_POOL_MAXSIZE,
```

Comment:
By adding more parameters please don't forget to adapt the documentation.

Reply:
Done ;-)